realm.gypi

```diff
@@ -175,7 +175,7 @@
         ]
       },
       "link_settings": {
-        "libraries": [ "-lrealm<(debug_library_suffix)" ],
+        "libraries": [ "-lrealm<(debug_library_suffix)", "-lrealm-parser<(debug_library_suffix)" ],
```

Comment:
These need to be linked in the reverse order so that GNU ld will find core symbols referenced by librealm-parser.a within librealm.a.